scripts/index-genenetwork

```diff
@@ -240,7 +240,7 @@ def build_rdf_cache(sparql_uri: str, query: str, remove_common_words: bool = Fal
     return smaller_cache
 
 
-def hash_rdf_graph(ttl_dir: pathlib.Path) -> str:
```

Comment:
I like the (arguably) nicer names. I now know (ideally) what the function should be doing without reading the code. Thanks. I should strive for this too.